src/NBitcoin/ConsensusOptions.cs

```diff
@@ -2,18 +2,26 @@
 {
     /// <summary>
     /// An extension to <see cref="Consensus"/> to enable additional options to the consensus data.
+    /// TODO: Make immutable.
     /// </summary>
     public class ConsensusOptions
     {
```

Comment:
I know the reordering looks painful but semantically this ordering makes much more sense.